### DISCUSSION TITLE
k8s服务重启后，怎样保证重新拉起后服务正常？

### DISCUSSION THREAD

**Discussion by dxsooo** [milvus-io/milvus#17100] [Q&A and General discussion]
公司运维需要进行k8s基座的升级维护（会中断k8s的服务），由于milvus是一个stateful set，想确定一下有没有一些机制或者操作（如提前手动关闭）来保证重新拉起后服务正常

当前milvus使用基于helm-chart（v3.0.23）启动，其中minio使用standalone并指定到预先创建出的pvc中

**Comment by LoveEachDay** [comment]
麻烦提供一下helm 安装的时候指定的参数。
milvus 本身的状态都维护在etcd, pulsar, minio 中。只要使用pvc 能够重新被挂载上对应的pod 上，服务重新拉起来没问题的。

**Reply by dxsooo** [reply]
`--set minio.mode=standalone --set minio.persistence.existingClaim=pvc-milvus-data-20220503`

只有这些，pulsar和etcd的pvc均由helm 自动创建，这里担心的是重启之后pvc挂不上
